File: src/main/java/com/telenordigital/prime/storage/ProductDescriptionCache.java
package com.telenordigital.prime.storage;

import com.telenordigital.prime.storage.entities.Product;

public interface ProductDescriptionCache {
    void  addTopupProduct(String sku, long noOfBytes);

    boolean isValidSKU(String sku);

    Product getProductForSku(String sku);
}

File: prime-api/src/main/kotlin/org/ostelco/prime/storage/legacy/ProductDescriptionCache.kt
package org.ostelco.prime.storage.legacy


import org.ostelco.prime.model.Product

interface ProductDescriptionCache {

    fun addTopupProduct(sku: String, noOfBytes: Long)

    fun isValidSKU(sku: String): Boolean

    fun getProductForSku(sku: String): Product?
}
